### ISSUE TITLE
role "vagrant" does not exist

### ISSUE THREAD

**Issue by danielgsims** [taigaio/taiga-scripts#38]
With a fresh vagrant install, I see the following error:

![screen shot 2015-11-02 at 9 56 02 am](https://cloud.githubusercontent.com/assets/695933/10884799/005b4a7c-8148-11e5-857e-fbb8182af486.png)

**Comment by superalex** [comment]
Hello @danielgsims what installation instructions did you follow?

**Comment by chaho** [comment]
no



chaho

From: Alejandro
Date: 2015-11-03 14:37
To: taigaio/taiga-scripts
Subject: Re: [taiga-scripts] role "vagrant" does not exist (#38)
Hello @danielgsims what installation instructions did you follow?

**Comment by superalex** [comment]
@chaho sorry but I don't understand your answer...

**Comment by danielgsims** [comment]
@superalex - I followed the vagrant installation instructions.

http://taigaio.github.io/taiga-doc/dist/setup-alternatives.html#setup-taiga-vagrant

I appreciate the help!

**Comment by superalex** [comment]
Hello @danielgsims ,

I just did a fresh vagrant install and everything is working ok...Could you try checking if you had any problem during the installation or destroying the vagrant machine and creating it again?

**Comment by danielgsims** [comment]
I did a fresh install and still getting the same issue. Here's the output during vagrant initialization.

http://git/gist/dsims1/82afe88e9f873fa38233

I see a couple of errors, the first one looks like it might be the problem. Do you have any insight into these?

http://git/gist/dsims1/82afe88e9f873fa38233#file-vagrant-response-L430-L448
http://git/gist/dsims1/82afe88e9f873fa38233#file-vagrant-response-L860-L862
http://git/gist/dsims1/82afe88e9f873fa38233#file-vagrant-response-L876-L878

**Comment by superalex** [comment]
Sorry but I can't see the links correctly...

**Comment by danielgsims** [comment]
I edited the links with some markdown, can you open them now?

**Comment by superalex** [comment]
No...

**Comment by danielgsims** [comment]
Sorry about that. Can you view the gist now?

**Comment by bameda** [comment]
Yes i see it. I fix the error with TEMPLATE_* in taiga-scripts and test it. This is my console log

https://gist.github.com/bameda/90749c8a720e4c0c2624

Can you try it again?:

```
rm -rf taiga-vagrant
rm -rf ~/VirtualBox\ VMs
git clone https://github.com/taigaio/taiga-vagrant.git
cd taiga-vagrant
vagrant up
```

**Comment by danielgsims** [comment]
I tried it again. I'm not seeing the same errors while installing, but I get the same 500 Operational Error.

Here's my output from vagrant up.

https://gist.github.com/danielgsims/ae57d799ce818d97652f

**Comment by bameda** [comment]
Your error is with grub-pc package
```
==> default: Errors were encountered while processing:
==> default:  grub-pc
==> default: E: Sub-process /usr/bin/dpkg returned an error code (1)
==> default:  * Restarting nginx nginx
==> default:    ...done.
```

In my setup this package is not installed, why? 

Are you remove the vbox image and the vagrant base box image before the reinstall process?

Try to reinstall grub-pc or reconfigure ```dpkg-reconfigure -a```. and run the taiga script again.

**Comment by danielgsims** [comment]
@bameda I must have had a problem with my trusty64 vagrant box. I had to delete my trusty64 entry within .vagrant.d. After re-downloading the box, it worked. I must have had a conflict with another box named trusty64 from an older installation. 

Thank you all for the help.

**Comment by bameda** [comment]
Great!! You are welcome
